core/src/test/java/org/apache/brooklyn/core/effector/http/HttpCommandEffectorHttpBinTest.java

```diff
@@ -38,42 +54,93 @@
 import com.google.common.collect.ImmutableMap;
 import com.jayway.jsonpath.JsonPath;
 
-public class HttpCommandEffectorIntegrationTest {
+public class HttpCommandEffectorHttpBinTest {
 
     final static Effector<String> EFFECTOR_HTTPBIN = Effectors.effector(String.class, "Httpbin").buildAbstract();
 
+    private TestHttpServer server;
+    private String serverUrl;
     private TestApplication app;
     private EntityLocal entity;
     
+    public static final class HttpBinRequestHandler implements HttpRequestHandler {
```

Comment:
Would it be simpler to use BetterMockWebServer here as in https://github.com/duncangrant/brooklyn-server/blob/1c064b284757ed0c794cbf8f3a6d65e34564af3a/core/src/test/java/org/apache/brooklyn/core/effector/http/HttpCommandEffectorTest.java#L68-L70